tests/Resources/Files/uninstall-model-serving.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+MODELMESH_PROJECT=${1:-"redhat-ods-applications"}
+INFERENCE_SERVICE_PROJECT=${2:-"mesh-test"}
+
+oc delete kfdef odh-modelmesh -n $MODELMESH_PROJECT --ignore-not-found=true &
```

Comment:
is `&` a mistake or desired?

Reply:
it's desired, without it the delete command hangs and I cannot move on to the patch one